### ISSUE TITLE
7tv support for twitch?

### ISSUE THREAD

**Issue by Tumppi066** [teststaybaka/danmage#6]
The extension works fine on twitch until 7tv loads, after that it's not showing anything. I assume this is because 7tv messes with the chat component?

**Comment by teststaybaka** [comment]
yep. I checked twitch, It completely replace the entire chat box with its own element `<main data-v-bab7a8f6="" class="seventv-chat-list" alternating-background="false">`, which is insane. Maybe I can look for that class name as well.

**Comment by Tumppi066** [comment]
If you could get it working that would be fantastic!

**Comment by teststaybaka** [comment]
Version 5.1.0 will support it. Being approved by Chrome Web Store.
